verif/big_core/tests/bubblesort_back_pressure.vh

```diff
@@ -1,15 +1,23 @@
 // generating non deterministic back pressure at non deterministic times
 
-delay(30);
-random = $urandom_range(1, 10);
-
-$display("Back pressure will be activated :%1d times", random);
-for(int i=0; i <random; i++) begin
-    #(50*random)  // deactivate back pressure time
-    $display("Back presure activated at: %t", $time);
-    force big_core_top.big_core_mem_wrap.DMemReady = 1'b0; 
-    #(10*random)  // activate back pressure time
-    $display("Back presure deactivated at: %t", $time);
-    release big_core_top.big_core_mem_wrap.DMemReady;
-end
+    random = $urandom_range(10, 100); // Set the number of back pressure events
+
+    // random = 7'd65; // Set the number of back pressure events
+    $display("Back pressure will be activated: %1d times", random);
+    for (i = 0; i < random; i++) begin
+        $display("Loop iteration: %d, Time: %t", i, $time);
+        repeat (random) @(posedge Clk); // Reduce the number for debugging
```

Comment:
What's the reduce The number for debug mean?

Reply:
The comment is not correct
The main reason is that I want to be sure every thing is synchronized after I finish with release. I saw at the waves and it work as expected. I saw sometimes when not using such lines it causes behavorial that I cant explain. I decide to use that lines every time before and after forse and release